app/models/route.rb

```diff
@@ -16,7 +16,8 @@ class Route < ActiveRecord::Base
 
     def self.search(search)
         if search
-            Busstop.find(:all, :conditions => ["\'busStopName\' LIKE ?", "%#{search}%"])
+            Busstop.find(:all, :conditions => ['busStopName LIKE ?', "%#{search}%"])
+            #Busstop.find(:all, :conditions => ["\'busStopName\' LIKE ?", "%#{search}%"])
```

Comment:
Line is too long. [89/80]<br>Incorrect indentation detected (column 12 instead of 10).<br>Missing space after #.